agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -139,7 +139,7 @@ static std::string get_sta_iface_from_hostap_iface(const std::string &hostap_ifa
     return sta_iface;
 }
 
-static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_slave_conf,
+static void fill_son_slave_config(const beerocks::config_file::sConfigSlave &beerocks_slave_conf,
```

Comment:
This is also a const-ref fix so please reuse the same commit message as in the first two.